TAScheduler/views_methods.py

```diff
@@ -193,28 +194,61 @@ def __init__(self, course_info):
         self.course_database = course_info
 
     def addInstructor(self, active_instr):
-        pass
+        if not isinstance(active_instr, InstructorObj):
+            raise TypeError("active_instr is not an instance of InstructorObj")
+        if InstructorToCourse.objects.filter(course=self.course_database).count() >= self.course_database.max_instructors:
```

Comment:
Good logic - but I don't think there's a "max_instructors" field in the course database (I think this should work if you replace it with "num_of_sections" which is what I did). 

Logic: I think, (not 100% sure), that if there's 6 sections allowed per course, then we allow at most 6 users-1 for each section-

 ... so this field is essentially the same thing IMO